client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -630,19 +642,15 @@ export class CheckoutThankYou extends Component<
 				} ) }
 			>
 				<PageViewTracker { ...this.getAnalyticsProperties() } title="Checkout Thank You" />
-				{ this.isDataLoaded() && isRedesignV2( this.props ) && (
-					<ConfettiAnimation delay={ 1000 } />
-				) }
-				{ isRedesignV2( this.props ) && this.props.selectedSite?.ID && (
-					<>
-						<QuerySitePurchases siteId={ this.props.selectedSite.ID } />
-						<MasterbarStyled
-							onClick={ () => page( `/home/${ this.props.selectedSiteSlug ?? '' }` ) }
-							backText={ translate( 'Back to dashboard' ) }
-							canGoBack={ true }
-							showContact={ true }
-						/>
-					</>
+				{ this.isDataLoaded() &&
+					isRedesignV2( this.props ) &&
+					! this.isBulkDomainTransfer( purchases ) && <ConfettiAnimation delay={ 1000 } /> }
```

Comment:
We're not having confetti for domain transfers 🥹 🎊 ?
(feel free to ignore this comment)

Reply:
This is a reaaaaally good question! 😄 I'm happy to bring it back!